src/web/routes/application/check/get.test.js

```diff
@@ -1,5 +1,41 @@
 const test = require('tape')
-const { getRowData, getLastStepPath, getFlattenedRowData } = require('./get')
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+
+const { getRowData, getFlattenedRowData } = require('./get')
+
+const getPreviousPath = sinon.spy()
+const { getLastNavigablePath } = proxyquire('./get', { '../common/get-previous-path': { getPreviousPath } })
+
+test('getLastNavigablePath returns path of last step if last step has not isNavigable function', (t) => {
```

Comment:
Typo `... if last step has not isNavigable ...` -> `... if last step has no isNavigable ...`

Reply:
Fixed typo

Reply:
Fixed typo.